### PR TITLE
[FLINK-17635][docs][table sql / api] Add documentation about view support

### PR DESCRIPTION
**Pull request by leonardBang**
## What is the purpose of the change

* We support Create/Drop/DESCRIBE view in 1.11. this pull request Add documentation about view support.

## Brief change log

  - *Add Create/Drop/DESCRIBE view section in SQL documentation*
  - *Translate corresponding Chinese documentation*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): ( no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/table/sql/drop.md [hunk 1]
```diff
@@ -143,6 +144,22 @@ Dropping a non-empty database triggers an exception. Enabled by default.
...
+DROP [TEMPORARY] VIEW  [ IF EXISTS ] view_name
+{% endhighlight %}
+
+Drop a view that has catalog and database namespaces. If the view to drop does not exist, an exception is thrown.
+
+**IF EXISTS**
+
+If the view does not exist, nothing happens.
+
+**TEMPORARY**
```
Comment (godfreyhe) [inline docs/dev/table/sql/drop.md:159]: move this section to the front of **IF EXISTS**

File: docs/dev/table/sql/drop.zh.md [hunk 2]
```diff
@@ -143,6 +144,22 @@ DROP DATABASE [IF EXISTS] [catalog_name.]db_name [ (RESTRICT | CASCADE) ]
...
+DROP [TEMPORARY] VIEW  [ IF EXISTS ] view_name
+{% endhighlight %}
+
+删除一个有 catalog 和数据库命名空间的视图。若需要删除的视图不存在，则会产生异常。
+
+**IF EXISTS**
+
+若视图不存在，则不会进行任何操作。
+
+**TEMPORARY**
```
Comment (godfreyhe) [inline docs/dev/table/sql/drop.zh.md:159]: ditto

File: docs/dev/table/sql/queries.md [hunk 3]
```diff
@@ -313,9 +313,29 @@ SHOW DATABASES;
...
 {% endhighlight %}
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Describe</strong><br>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
+      </td>
+      <td>
+			<p>Describe the given table.</p>
```
Comment (godfreyhe) [inline docs/dev/table/sql/queries.md:329]: explain more about `Describe` operation ? such as `Describe the schema of the given table.`

File: docs/dev/table/sql/queries.zh.md [hunk 4]
```diff
@@ -313,9 +313,29 @@ SHOW DATABASES;
...
 {% endhighlight %}
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Describe</strong><br>
+        <span class="label label-primary">批处理</span> <span class="label label-primary">流处理</span>
+      </td>
+      <td>
+			<p>描述给定表的元数据.</p>
```
Comment (godfreyhe) [inline docs/dev/table/sql/queries.zh.md:329]: currently, only support describing table schema

File: docs/dev/table/sql/create.md [hunk 5]
```diff
@@ -351,6 +352,25 @@ The key and value of expression `key1=val1` should both be string literal.
 
 {% top %}
 
+## CREATE VIEW
+{% highlight sql %}
+CREATE [TEMPORARY] VIEW [IF NOT EXISTS] view_name
```
Comment (godfreyhe) [inline docs/dev/table/sql/create.md:357]: CREATE [TEMPORARY] VIEW [IF NOT EXISTS] [catalog_name.][db_name.]view_name

File: docs/dev/table/sql/create.zh.md [hunk 6]
```diff
@@ -344,6 +345,25 @@ CREATE DATABASE [IF NOT EXISTS] [catalog_name.]db_name
 
 {% top %}
 
+## CREATE VIEW
+{% highlight sql %}
+CREATE [TEMPORARY] VIEW [IF NOT EXISTS] view_name
```
Comment (godfreyhe) [inline docs/dev/table/sql/create.zh.md:350]: ditto

File: docs/dev/table/sql/drop.md [hunk 7]
```diff
@@ -143,6 +144,22 @@ Dropping a non-empty database triggers an exception. Enabled by default.
 
 Dropping a non-empty database also drops all associated tables and functions.
 
+## DROP VIEW
+
+{% highlight sql %}
+DROP [TEMPORARY] VIEW  [ IF EXISTS ] view_name
```
Comment (godfreyhe) [inline docs/dev/table/sql/drop.md:150]: ditto

File: docs/dev/table/sql/drop.zh.md [hunk 8]
```diff
@@ -143,6 +144,22 @@ DROP DATABASE [IF EXISTS] [catalog_name.]db_name [ (RESTRICT | CASCADE) ]
 
 删除一个非空数据库时，把相关联的表与函数一并删除。
 
+## DROP VIEW
+
+{% highlight sql %}
+DROP [TEMPORARY] VIEW  [ IF EXISTS ] view_name
```
Comment (godfreyhe) [inline docs/dev/table/sql/drop.zh.md:150]: ditto

File: docs/dev/table/sql/queries.zh.md [hunk 9]
```diff
@@ -326,11 +326,11 @@ SHOW VIEWS;
         <span class="label label-primary">批处理</span> <span class="label label-primary">流处理</span>
       </td>
       <td>
-			<p>描述给定表的元数据.</p>
+			<p>描述给定表的模式.</p>
```
Comment (godfreyhe) [inline docs/dev/table/sql/queries.zh.md:329]: use `Schema` instead of  `模式` ?

Comment (leonardBang) [reply docs/dev/table/sql/queries.zh.md:329]: I’m fine to use **Schema** or **模式**, because terminology **schema** can translate to **模式** in DB area.

Comment (leonardBang) [reply docs/dev/table/sql/queries.zh.md]: Using Schema...

**Comment by godfreyhe** [comment]
I think it's better we can explain more about the behavior of dependencies maintaining, such as: does `drop a table` operation affect a existing view ? (see https://cwiki.apache.org/confluence/display/FLINK/FLIP-71%3A+E2E+View+support+in+FLINK+SQL)

**Pull request merged**